src/configs/git_branch.rs

```diff
@@ -1,26 +1,21 @@
-use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+use crate::config::{ModuleConfig, RootModuleConfig};
 
-use ansi_term::{Color, Style};
 use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct GitBranchConfig<'a> {
-    pub symbol: SegmentConfig<'a>,
     pub truncation_length: i64,
     pub truncation_symbol: &'a str,
-    pub branch_name: SegmentConfig<'a>,
-    pub style: Style,
+    pub format: &'a str,
```

Comment:
It is recommended not to remove `symbol` and `style` as per the discussion in https://github.com/starship/starship/discussions/1116#discussioncomment-6579 .

Reply:
I have restored `symbol` and `style` to match the new conventions.